File: Hashing/voting.cpp
void winner(string arr[],int n)
{
    int max = -1;
    int index;
    map<string,int> s;
    for(int i=0;i<n;i++){
        s[arr[i]]+=1;
        if(s[arr[i]]>max){
            max=s[arr[i]];
        }
    }
    for(auto it=s.begin();it!=s.end();it++){
        if(it->second==max){
            cout<<it->first<<" "<<it->second;
            break;
        }
    }
}

File: Hashing/voting.py
import math
#Complete this function
def winner(arr,n):
    s={}
    ans=[]
    max = -math.inf
    for x in arr:
        try:
            s[x]+=1
        except:
            s[x]=1
        if(s[x]>max):
            max = s[x]
    # s= sorted(key= lambda x:x[0])
    for x in s:
        if(s[x]==max):
            ans.append(x)
    ans.sort()
    print(str(ans[0])+" "+str(s[ans[0]]),end=" ")